src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+function growthTable(table) {
+  document.addEventListener('click', (evenFunc) => {
+    let rowsCount = table.children.length;
+    let columnCount = table.children[0].children.length;
+
+    const classButton = evenFunc.target.className;
+
+    switch (classButton) {
+      case 'append-row button' : {
+        table.append(table.children[0].cloneNode(true));
+        rowsCount = rowsCount + 1;
+        break;
+      }
+
+      case 'remove-row button' : {
+        table.children[0].remove();
+        rowsCount = rowsCount - 1;
+        break;
+      }
+
+      case 'append-column button' : {
+        for (const i of table.children) {
+          i.append(document.createElement('td'));
+        }
+        columnCount++;
+        break;
+      }
+
+      case 'remove-column button': {
+        for (const i of table.children) {
+          i.children[0].remove();
+        }
+        columnCount--;
+        break;
+      }
+    }
+
+    if (rowsCount <= 9) {
+      document.querySelector('.append-row').removeAttribute('disabled');
```

Comment:
this line is duplicated a lot, move it into helper function
`
 const removeDisabled = (class) => document.querySelector(class).removeAttribute('disabled');`